src/app/components/OptimizelyPageMetrics/index.test.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { render } from '../react-testing-library-with-providers';
+import OptimizelyPageMetrics from '.';
+
+jest.mock('./PageCompleteTracking', () => () => (
+  <div data-testid="page-complete-tracking" />
+));
+jest.mock('./ScrollDepthTracking', () => () => (
+  <div data-testid="scroll-depth-tracking" />
+));
+jest.mock('./PageViewTracking', () => () => (
+  <div data-testid="page-view-tracking" />
+));
+
+describe('OptimizelyPageMetrics', () => {
+  it('returns null when isAmp is true', () => {
+    const { container } = render(
+      <OptimizelyPageMetrics trackPageView trackPageDepth trackPageComplete />,
+      { isAmp: true },
+    );
+    expect(container.firstChild).toBeNull();
+  });
+
+  it('renders no tracking components by default when all tracking flags are false', () => {
+    render(<OptimizelyPageMetrics />, { isAmp: false });
```

Comment:
The default is already false, so can be removed: https://github.com/bbc/simorgh/blob/latest/src/app/components/react-testing-library-with-providers.tsx#L41-L42 (implement for all tests below)
```suggestion
    render(<OptimizelyPageMetrics />);
```

Reply:
Ahhh sorry I missed this one before - have updated